CHANGELOG.md

```diff
@@ -37,6 +37,8 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+* [\#5531](https://github.com/cosmos/cosmos-sdk/issues/5531) Added amount event to MsgMultiSend, emitted for each output.
```

Comment:
Miguel, can you add this line at the end of the `Features` stanza please https://github.com/cosmos/cosmos-sdk/pull/5557/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eL131?

Reply:
Done